### PR TITLE
GH actions integration

### PR DESCRIPTION
**Pull request by omesser**
- CI workflow includes ci lint and tests and a release process
- Keeping travis integration - for integration and benchmark - this is because GH actions cannot use custom secrets for PR workflows
- Removing Jenkinsfile tsdbctl release logic (will now only be pushed to github, not artifactory)
- Changing `README.md` badge to the new github workflow CI status
- Minor `Makefile` changes
- Moved CI test related v3io configs into `/test` dir (can be further moved into `/test/integration/` and `test/benchmark/` ?)

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    - uses: actions/checkout@v2
+
+    - name: Lint
+      run: make lint
+
+    - name: Test short
+      run: make test
+
+# Sisabled until GH actions will have a solution to using secret on PR triggered workflows.
```
Comment (gtopper) [inline .github/workflows/ci.yaml:58]: Sisabled -> Disabled

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/v3io/v3io-tsdb.svg?branch=master)](https://travis-ci.org/v3io/v3io-tsdb)
+[![Build Status](https://github.com/v3io/v3io-tsdb/workflows/CI/badge.svg)](https://github.com/v3io/v3io-tsdb/actions)
```
Comment (gtopper) [inline README.md:1]: I guess we do want to keep travis here then, alongside github.

File: Makefile [hunk 3]
```diff
@@ -4,8 +4,8 @@ ifeq ($(GIT_BRANCH),)
 	GIT_BRANCH="N/A"
 endif
 
-ifneq ($(TRAVIS_TAG),)
-	GIT_REVISION := $(TRAVIS_TAG)
+ifneq ($(TSDB_TAG),)
```
Comment (gtopper) [inline Makefile:7]: Where is `TSDB_TAG` defined?

Comment (omesser) [reply Makefile:7]: Should be TSDB_LABEL, fixed

**Pull request merged**